### DISCUSSION TITLE
Welcome to custom-tables Discussions!

### DISCUSSION THREAD

**Discussion by joomlaboat** [joomlaboat/custom-tables#65] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by joomlaboat** [comment]
Hi. My name is Ivan, I added this Custom Tables Discussions to support users and improve the usability of the component.

**Comment by HarkeyGroup** [comment]
Hi Ivan, when creating a menu item I get this error: An error has occurred. 0 Undefined constant "CUSTOMTABLES_LIBRARIES_PATH"
Joomla 4.4.3 PHP 8.3
I would like to see this work before upgrading to Pro.

**Reply by joomlaboat** [reply]
Hi, didn't see this message. Please try to update the component - an issue has been solved long ago.
